assets/src/application/ui/layout/entityList/filterBar/style.scss

```diff
@@ -78,19 +68,6 @@ $filter-btn: '.ee-button.ee-filter-bar__btn';
 			width: 100%;
 		}
 
-		.ant-input {
-			padding: var(--ee-padding-micro) var(--ee-padding-tiny);
-
-			&:hover,
-			&:focus {
-				border-color: unset;
-				border-right-width: 2px !important;
-				border: 2px solid var(--ee-color-primary-low-contrast);
-				box-shadow: none;
-				outline: none;
-			}
-		}
-
 		.ant-select,
 		.ant-select:not(.ant-select-disabled),
 		.ant-select-single:not(.ant-select-customize-input) {
```

Comment:
assuming this will get chopped in another PR

Reply:
yes, didn't wanted to make diff too big